### PR TITLE
Replacing pancake swap ui code with celeste ui code

### PR DESCRIPTION
**Pull request by saltoriousSIG**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does
Replacing pancake swap code with celese ui code

### CODE REVIEW

**Comment by crisog** [comment]
The `aragon-ui` folder should go in the `.gitignore` and ideally it would be placed in the `./public` directory.

These assets are copied while building to production and should not be included in the source code.

**Comment by crisog** [comment]
The `aragon-ui` folder should be placed in the `./public` directory, in fact, that is where the `.gitignore` assumes it is.

**Comment by saltoriousSIG** [comment]
Just removed it.

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { RequestQueueProvider } from './providers/RequestQueue'
+import { WalletProvider } from './providers/Wallet'
+
+function App() {
+  return (
+    <WalletProvider>
+      <HashRouter>
+        <ActivityProvider>
+          <Main
+            assetsUrl="./aragon-ui/"
```
Comment (crisog) [inline src/App.js:24]: This seems to be the default configuration, the folder is no longer here

Comment (crisog) [inline src/App.js]: This seems to be the default configuration, should be updated.

**Comment by saltoriousSIG** [comment]
Got it, updated the path

**Comment by famole** [comment]
can you point the pr to celeste-fe branch merge it and then create a new pr pointing to master? I need to clone the new code to my actual fork

**Pull request merged**